benches/ipc.rs

```diff
@@ -0,0 +1,73 @@
+#![allow(clippy::identity_op)]
+use criterion::{criterion_group, criterion_main, Criterion};
+use ipc_channel::ipc;
+
+/// Allows doing multiple inner iterations per bench.iter() run.
+///
+/// This is mostly to amortise the overhead of spawning a thread in the benchmark
+/// when sending larger messages (that might be fragmented).
+///
+/// Note that you need to compensate the displayed results
+/// for the proportionally longer runs yourself,
+/// as the benchmark framework doesn't know about the inner iterations...
+const ITERATIONS: usize = 1;
```

Comment:
It would be possible to eliminate, but I wanted to limit the scope of this PR to make transition as close as possible as this is not as easy as porting to criterion.

https://bheisler.github.io/criterion.rs/book/user_guide/timing_loops.html#iter_custom

Reply:
Okay. Sure. No problem.

Reply:
Opened https://github.com/servo/ipc-channel/issues/353.